config/sources/families/meson-g12b.conf

```diff
@@ -1,24 +1,25 @@
 source "${BASH_SOURCE%/*}/include/meson64_common.inc"
 ASOUND_STATE="asound.state.meson64"
-UBOOT_TARGET_MAP=';;sd_fuse/u-boot.bin'
-[[ ${BRANCH} == legacy ]] && SERIALCON=ttyS0
-UBOOT_USE_GCC='< 4.9'
-UBOOT_TOOLCHAIN2="arm-none-eabi-:< 5.0"
-UBOOT_COMPILER="aarch64-none-elf-"
 
-BOOTSOURCE='https://github.com/hardkernel/u-boot.git'
-BOOTBRANCH='branch:odroidg12-v2015.01'
-BOOTPATCHDIR='u-boot-odroid'
-BOOTDIR='u-boot-odroid'
+[[ ${BRANCH} == legacy ]] && SERIALCON=ttyS0
 
-UBOOT_TARGET_MAP=';;sd_fuse/u-boot.bin'
-BOOTSCRIPT="boot-odroid-n2.ini:boot.ini"
+# Setting these values in the board.conf is useless, they're overwritten by the meson64_common.inc include above.
```

Comment:
We used to re-read board values, but this principle could be broken.

Reply:
Yep, re-reading board.conf after sourcing the family could alleviate the pain. The fact is too many places test for $board, all over the codebase, family.conf being one of them. A reorg is in order.